src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -130,7 +130,7 @@ private int ExploreTests(TestPackage package, TestFilter filter)
                 }
             }
 
-            return ConsoleRunner.OK;
+            return OK;
```

Comment:
Clearly the class name is not needed. Equally clearly the author wanted it there... for clarity.